setup.py

```diff
@@ -94,6 +94,7 @@
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
         'paramiko>=1.15.2',
+        'six>=1.10',
```

Comment:
I know about the existence of this module. And there is a more convenient unit for writing compatible code https://pypi.python.org/pypi/future . But I think all of these modules are not for this project, it makes no sense to use one constant from module `six` so it is clear that we have to write. 

Use a third-party module in this case is overhead.

I need to write about it in the comments, so that future contributors not tried to add this dependency.